packages/client/src/Store/RootReducer.ts

```diff
@@ -0,0 +1,23 @@
+import { CurrentModal, DBLoadingState, ExportItems, GameInfo, Messages, UmpireMenu } from "@serge/custom-types"
+import { messagesReducer } from "../ActionsAndReducers/dbMessages/messages_Reducer"
+import { exportItems } from "../ActionsAndReducers/ExportItems/ExportItems_Reducer"
+import { loadingDbReducer } from "../ActionsAndReducers/loadingDb_Reducer"
+import { currentModal } from "../ActionsAndReducers/Modal/Modal_Reducer"
+import { notificationReducer } from "../ActionsAndReducers/Notification/Notification_Reducer"
+import { gameInfo } from "../ActionsAndReducers/sergeInfo/sergeInfo_Reducer"
+import { umpireMenuReducer } from "../ActionsAndReducers/UmpireMenu/umpireMenu_Reducer"
+
```

Comment:
Hello @lebaphi - there is a big change here, but I can't quite see what it happening.  Could you have another go at explaining it please?

Aah, I think I've got it.  The reducers are written in JS rather than TS.   Right?

Reply:
yep, as you can see all all our reducers were written in JS so they don't have a type for it. I create this class to wrap the old reducer with new reducer (in TS file) then we can have a type for it.